glance/server.sls

```diff
@@ -79,6 +79,27 @@ glance_install_database:
   - require:
     - service: glance_services
 
+{%- if server.get('image_cache', {}).get('enabled', False) %}
+glance_cron_glance-cache-pruner:
```

Comment:
These cron jobs will be deployed on all control nodes. What will happen if they will run 3 times at once?